### ISSUE TITLE
Execution fails on OSX El Capitan

### ISSUE THREAD

**Issue by pchaganti** [butterproject/butter#30]
```bash
hades:~/Dev/butter[master]$ grunt build

Running "injectgit" task

Running "bower_clean" task

Running "shell:themes" (shell) task
Submodule 'src/app/styl/third_party' (https://github.com/butterproject/butter-desktop-themes.git) registered for path 'src/app/styl/third_party'

Cloning into 'src/app/styl/third_party'...
Submodule path 'src/app/styl/third_party': checked out '8564c83184fac53127c0af1aebdc9e98a04a00f4'

Running "clean:css" (clean) task
>> 0 paths cleaned.

Running "stylus:official" (stylus) task
>> 5 files created.

Running "stylus:third_party" (stylus) task
>> 14 files created.

Running "nwjs:src" (nwjs) task
Using v0.12.3
Create cache folder in /Users/hades/Dev/butter/cache/0.12.3
Downloading: http://dl.nwjs.io/v0.12.3/nwjs-v0.12.3-osx-ia32.zip
Create cache folder in /Users/hades/Dev/butter/cache/0.12.3
Downloading: http://dl.nwjs.io/v0.12.3/nwjs-v0.12.3-osx-x64.zip
  downloading [====================] 100% 0.0s

Create release folder in /Users/hades/Dev/butter/build/Butter/osx32
Create release folder in /Users/hades/Dev/butter/build/Butter/osx64
chmod 33188 on /Users/hades/Dev/butter/build/Butter/osx32/Butter.app/Contents/Resources/nw.icns failed after copying, ignoring
chmod 33188 on /Users/hades/Dev/butter/build/Butter/osx64/Butter.app/Contents/Resources/nw.icns failed after copying, ignoring
>> NW.js app created.

Running "shell:setexecutable" (shell) task
chmod: build/releases/Butter/mac/Butter.app: No such file or directory
chmod: build/releases/Butter/linux*/Butter/Butter: No such file or directory

Done, without errors.

hades:~/Dev/butter[master]$ grunt start
Running "start" task

Running "exec:mac" (exec) task
>> Exited with code: 127.
>> /bin/sh: build/cache/mac/0.12.3/nwjs.app/Contents/MacOS/nwjs: No such file or directory
Warning: Task "exec:mac" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by MrAdder** [comment]
Try changing line 233-235 in gruntfile.js with             
mac: {
                cmd: 'cache/mac/<%= nwjs.options.version %>/nwjs.app/Contents/MacOS/nwjs .'
            }

after changing that line try grunt start again if it works let me know

**Comment by edospadoni** [comment]
file Gruntfile.js line 234.
change this: 
`cmd: 'build/cache/mac/<%= nwjs.options.version %>/nwjs.app/Contents/MacOS/nwjs .'`
to this:
`cmd: 'cache/<%= nwjs.options.version %>/osx64/nwjs.app/Contents/MacOS/nwjs .'`

**Comment by xaiki** [comment]
fixed in c38db580dc21c8db94370e631c3c10776383309e
